go.mod

```diff
@@ -1,26 +1,18 @@
 module storj.io/storj
 
 require (
-	9fans.net/go v0.0.0-20180727211846-5d4fa602e1e8 // indirect
-	cloud.google.com/go v0.26.0 // indirect
-	contrib.go.opencensus.io/exporter/stackdriver v0.6.0 // indirect
-	github.com/Azure/azure-sdk-for-go v19.1.0+incompatible // indirect
-	github.com/Azure/go-autorest v10.15.2+incompatible // indirect
-	github.com/alecthomas/gometalinter v2.0.6+incompatible // indirect
-	github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf // indirect
+	github.com/BurntSushi/toml v0.3.0 // indirect
+	github.com/Shopify/toxiproxy v2.1.3+incompatible // indirect
 	github.com/alicebob/gopher-json v0.0.0-20180125190556-5a6b3ba71ee6 // indirect
 	github.com/alicebob/miniredis v0.0.0-20180830063239-35be932987bc
-	github.com/aliyun/aliyun-oss-go-sdk v0.0.0-20180615125516-36bf7aa2f916 // indirect
-	github.com/anacrolix/missinggo v0.0.0-20180525074307-ca16e9f398a9
-	github.com/anacrolix/sync v0.0.0-20180525101250-3870fa5b90c5
-	github.com/anacrolix/utp v0.0.0-20180219060659-9e0e1d1d0572
+	github.com/anacrolix/missinggo v0.0.0-20180525074307-ca16e9f398a9 // indirect
+	github.com/anacrolix/sync v0.0.0-20180525101250-3870fa5b90c5 // indirect
+	github.com/anacrolix/utp v0.0.0-20180219060659-9e0e1d1d0572 // indirect
+	github.com/armon/go-metrics v0.0.0-20180713145231-3c58d8115a78 // indirect
 	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 // indirect
 	github.com/boltdb/bolt v1.3.1
-	github.com/ccding/go-stun v0.0.0-20171206150302-d9bbe8f8fa7b
+	github.com/ccding/go-stun v0.0.0-20171206150302-d9bbe8f8fa7b // indirect
 	github.com/cheggaaa/pb v1.0.5-0.20160713104425-73ae1d68fe0b // indirect
-	github.com/cloudfoundry/gosigar v1.1.0
```

Comment:
i take it go mod tidy doesn't consider other platforms and you ran this on Windows or something? i think Linux still pulls this in